public/style.css

```diff
@@ -0,0 +1,305 @@
+  --space-third: 0.9rem;
+}
+
+/* general styling pages */
+
+* {
+  box-sizing: border-box;
+  font-family: var(--poppins);
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: grid;
+  width: 100vw;
+  min-height: 100vh;
+  background: var(--gradient-four);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+main {
+  place-self: center;
+  min-height: 100vh;
+  width: 80%;
+}
+
+/* nav styling */
+
+header button {
+  position: fixed;
+  display: grid;
+  place-items: center;
+  top: 5px;
+  left: 5px;
+  height: 50px;
+  width: 50px;
+  background-color: var(--menu);
+  border: none;
+  padding: var(--space-small);
+  border-radius: 6px;
+  transition: all 0.3s ease-out;
+}
+
+nav {
+  display: none;
+  position: fixed;
+  top: 0px;
+  transition: all 0.3s ease-in-out;
+}
+
+nav ul {
+  display: flex;
+  flex-direction: row;
+  width: 100vw;
+}
+
+nav li {
+  display: grid;
+  place-items: center;
+  width: 100%;
+  height: 105px;
+  list-style: none;
+  padding-top: var(--space-small);
+  transition: all 0.3s ease-in;
+}
+
+nav li:hover {
+  width: 120%;
+}
+
+nav a {
+  font-size: 0.9rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-decoration: none;
+  transition: all 0.2s ease-in-out;
+}
+
+nav a:visited {
+  color: var(--primary-one);
+}
+
+nav img {
+  width: 45px;
+  height: 45px;
+}
+
+nav .home {
+  background-color: var(--home);
+}
+
+nav .lessons {
+  background-color: var(--lessons);
+}
+
+nav .stories {
+  background-color: var(--stories);
+}
+
+nav .playlists {
+  background-color: var(--playlists);
+}
+
+nav .profile {
+  background-color: var(--profile);
+}
+
+nav li:hover img {
+  transition: all 0.2s ease-in-out;
+  scale: 1.05;
+}
+
+/* hamburger svg animation */
+
+.hamburger rect {
+  transition: all 0.25s ease-in-out;
+}
+
+svg .rect-one {
+  transform-origin: right;
+}
+
+svg .rect-three {
+  transform-origin: left;
+}
+
+.cross .rect-one {
+  transform: rotate(-45deg);
+  translate: 5px -20px;
+}
+
+.cross .rect-three {
+  transform: rotate(45deg);
+  translate: 35px -55px;
+}
+
+.cross .rect-two {
+  opacity: 0;
+  translate: -200px;
+}
+
+/* ANIMATIONS */
+
+nav {
+  animation: show-menu 0.3s ease-out;
+}
+
+@keyframes show-menu {
+  from {
+    opacity: 0;
+    translate: 0px -200px;
+  }
+  to {
+    opacity: 1;
+    translate: 0 0;
+  }
+}
+
+/* menu list items hover anim for the images */
+nav li:hover img {
+  animation: wobble 1.5s ease-in-out alternate infinite;
+}
+
+@keyframes wobble {
+  from {
+    rotate: -2deg;
+  }
+  to {
+    rotate: 2deg;
+  }
+}
+
+/* menu list item icon anim for when it appears */
+nav img {
+  animation: show-img 0.7s ease-out 1;
+}
+
+/* TODO commented out bc it also does this after hovering and idk how to remove it */
+/* @keyframes show-img {
+  from {
+    opacity: 0;
+    scale: 0.1;
+  }
+  to {
+    opacity: 1;
+    scale: 1;
+  }
+}  */
+
+/* toggle classes */
+
+.show {
+  display: block;
```

Comment:
_display:block_ is niet animeerbaar. Wellicht kan je met een ander css property een meer pleasurable UI maken?